core/domain/opportunity_services.py

```diff
@@ -146,26 +145,11 @@ def _create_exploration_opportunity_summary(topic, story, exploration):
         ExplorationOpportunitySummary. The exploration opportunity summary
         object.
     """
-
-    audio_language_codes = set(
-        language['id'] for language in constants.SUPPORTED_AUDIO_LANGUAGES)
-
-    complete_translation_languages = set(
+    language_codes_needing_voice_artists = set(
         exploration.get_languages_with_complete_translation())
-
-    incomplete_translation_language_codes = (
-        audio_language_codes - complete_translation_languages)
-    language_codes_needing_voice_artists = complete_translation_languages
-
-    if exploration.language_code in incomplete_translation_language_codes:
-        # Removing exploration language from incomplete translation
-        # languages list as exploration does not need any translation in
-        # its own language.
-        incomplete_translation_language_codes.discard(
-            exploration.language_code)
-        # Adding exploration language to voiceover required languages
-        # list as exploration can be voiceovered in it's own language.
-        language_codes_needing_voice_artists.add(exploration.language_code)
+    # Add exploration language to voiceover required languages list as an
+    # exploration can be voiceovered in its own language.
+    language_codes_needing_voice_artists.add(exploration.language_code)
```

Comment:
If the voiceover is complete, then should this still be added?

Reply:
Ah, good catch. Reverted to existing logic.